style.css

```diff
@@ -414,3 +415,209 @@
     background-color: #6070ff;
   }
 }
+
+@media screen and (min-width: 768px) {
+  #header {
+    height: 90px;
+    width: 100%;
+    padding-bottom: 40px;
+  }
+
+  .logo {
+    padding-left: 155px;
+    padding-top: 10px;
+  }
+
+  .nav {
+    --i1: scale(0);
+    --i2: scale(0);
+  }
+
+  .nav:has(.side-bar:target) {
+    --i1: scale(0);
+    --i2: scale(0);
+    --clip: inset(0 0 0 0);
+  }
+
+  .nav-container {
+    width: 100%;
+    margin: 0 auto;
+    display: flex;
+  }
+
+  .nav-menu {
+    display: none;
+  }
+
+  .nav-menu-second {
+    display: none;
+  }
+
+  .side-bar {
+    position: relative;
+    width: 100%;
+    background-color: #fff;
+    clip-path: none;
+    transition: none;
+    display: flex;
+    margin-right: 155px;
+  }
+
+  .side-bar li {
+    padding-right: 32px;
+    display: flex;
+    padding-bottom: 10px;
+  }
+
+  .side-bar li a {
+    color: rgba(52, 69, 99, 1);
+    font-size: 15px;
+  }
+
+  .social-container li a {
+    font-size: 16px;
+  }
+
+  .side-bar li a:hover {
+    color: rgba(52, 69, 99, 0.638);
+  }
+
+  .section-p1 {
+    background-image: url('img/Header-bg1.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top center;
+    height: 590px;
+    border-bottom: 1px solid rgba(223, 225, 230, 1);
+  }
+
+  .section-p1 .div1 {
+    margin-top: 90px;
+    margin-left: 360px;
+  }
+
+  .div1 h2 {
+    font-size: 48px;
+    width: 30%;
+    line-height: 60px;
+  }
+
+  .div1 p {
+    font-size: 20px;
+    width: 50%;
+  }
+
+  .section-p1 .div2 {
+    margin-left: 90px;
+    margin-left: 360px;
+  }
+
+  .section-p2 {
+    width: 100%;
+  }
+
+  .card {
+    width: 1156px;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    border: 1px solid rgba(223, 225, 230, 1);
+  }
+
+  .card-multi {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .card .img-project {
+    width: 100%;
+    height: 100%;
+    padding: 24px;
+  }
+
+  .description-project h3 {
+    font-size: 40px;
+    padding-bottom: 20px;
+  }
+
+  .multi {
+    padding-left: 50px;
+  }
+
+  .multi h3 {
+    width: 60%;
+    line-height: 50px;
+    padding-right: 10px;
+  }
+
+  .frame {
+    justify-content: space-evenly;
+  }
+
+  .section-p3 {
+    flex-direction: row;
+    height: 668px;
+    align-items: center;
+  }
+
+  .descript-social {
+    width: 30%;
+    margin-bottom: 100px;
+  }
+
+  .description1 > p {
+    width: 500px;
+  }
+
+  .description2 button {
+    margin-left: 26px;
+  }
+
+  .languages {
+    width: 40%;
+    height: 500px;
+  }
+
+  .dropdown-content {
+    display: flex;
+    justify-content: space-evenly;
+  }
+
+  .dropdown-content li {
+    border: none;
+    padding: 5px;
+    background: #f7f7f9;
+    margin-bottom: 30px;
+  }
+
+  .dropdown-content li img {
+    padding-right: 8px;
+  }
```

Comment:
- [ ] Kindly display each element in the dropdown in a column and remove the bottom line to follow the Figma design as requested in the project requirement.
### Your design :point_down: 
![Screenshot from 2023-05-04 11-17-39](https://user-images.githubusercontent.com/91142432/236189227-0efd3a68-61d7-4d8b-b691-53fa0de3817f.png)
### What it is supposed to look like:point_down:
![Screenshot from 2023-05-04 11-18-39](https://user-images.githubusercontent.com/91142432/236189474-6ba27b86-4a9d-4599-ad9d-5b0e0b75e83f.png)